### DISCUSSION TITLE
When Adding className React-Dropdown-tree-select css has been changed

### DISCUSSION THREAD

**Discussion by Poojatilekar** [dowjones/react-dropdown-tree-select#496] [Q&A]
hello,
Please find attached  screenshot  when adding className="fas fa-folder".
![image](https://user-images.githubusercontent.com/72187397/122897049-43f2a600-d367-11eb-8ae0-4c55df9e029e.png)

but when Selection on checkbox the css has been changed for more reference find out attached screenshot.
![image](https://user-images.githubusercontent.com/72187397/122897256-743a4480-d367-11eb-86c1-3b2908ad7faa.png)

**Comment by mrchief** [comment]
I'd suggest to take a look at https://dowjones.github.io/react-dropdown-tree-select/#/story/with-bootstrap-styles that uses FontAwesome to render icons on the nodes. 

When you use external libs like fontawesome, you need to understand how they are modifying the CSS and then you'd need to supply your own styles to compensate for any odd behaviors. That and then I'd also look at your own CSS to see what causes the inline behavior. The default styles are very basic so that you can easily target/override them without much `!important` hacks.

**Reply by Poojatilekar** [reply]
![image](https://user-images.githubusercontent.com/72187397/123041815-7e198180-d413-11eb-8426-dcc2965b4e6d.png)
I have checked Code but  fontAwesome  is not present. Can you explain me in detail. And also let me know which css property need to modify.

**Reply by mrchief** [reply]
FA is added via "External Resources". Setting this enables FA

```
font: normal normal normal 12px/1 FontAwesome;
```

and then you pick the corresponding glyph by saying `content: "\f067";` in your css classes
